lib/test/Test/Pos/Block/Logic/VarSpec.hs

```diff
@@ -55,6 +55,8 @@ import           Test.Pos.Helpers             (blockPropertySpec)
 import           Test.Pos.Util                (HasStaticConfigurations, splitIntoChunks,
                                                stopProperty, withStaticConfigurations)
 
+
+-- stack test cardano-sl --fast --test-arguments "-m Test.Pos.Block.Logic.Var"
```

Comment:
Do we actually need this comment?

Reply:
We don't _need_ it, but I prefer if I don't have to think about how to run a specific test. It's my preference though.